api/nodes/serializers.py

```diff
@@ -375,3 +377,57 @@ class Meta:
     @staticmethod
     def get_id(obj):
         return '{}:{}'.format(obj.node._id, obj.provider)
+
+class NodeAlternativeCitationSerializer(JSONAPISerializer):
+
+    id = IDField(source="_id", read_only=True)
```

Comment:
I think you also need `type = TypeField()` here to be consistent with our API spec.